### DISCUSSION TITLE
How to update locales offline?

### DISCUSSION THREAD

**Discussion by MahdiNajafzadeh** [requarks/wiki#6905] [Help / Questions] [answered]
hello ! 

i try to Update locales from update in locales table in database 
update in database is successfully but dont refresh in client ( or view ) 
Is this data cache somewhere?

The reason I used this method was that I did not want to connect the server to the Internet
But once I connected to the internet, it didn't change and it didn't update!

![Screenshot from 2023-12-26 22-47-06](https://github.com/requarks/wiki/assets/86781212/2b6f6bb7-546f-480b-b9c7-d19f694255bc)
![image](https://github.com/requarks/wiki/assets/86781212/ca36b796-98d6-4c8c-8d88-e7541d90cbe4)

i copy this data from a test server and dont have this problem ! 

test server : 

![image](https://github.com/requarks/wiki/assets/86781212/bb155db5-5425-438e-825d-af3ec9f28a7a)

how can i fix this problem !?

**Comment by NGPixel** [comment, marked as answer]
You need to clear your browser cache and restart Wiki.js in order to load the strings from the DB.
